website/views/shared/modals/death.jade

```diff
@@ -1,16 +1,40 @@
-//div(modal='user.stats.hp <= 0', options='{backdrop:true, keyboard:false, backdropClick:false}')
-script(type='text/ng-template', id='modals/death.html')  
-  .modal-body.death-modal
-    .container-fluid
-      .row
-        .col-md-3
-          figure
-            .GrimReaper
-        .col-md-9
-          h2=env.t('youDied')
-      .row
-        .col-md-3
-          p
-            a.btn.btn-danger.btn-lg(ng-click='user.ops.revive({}); $close()')=env.t('continue')
-        .col-md-9
-          p=env.t('dieText')
+include ../avatar/generated_avatar
+
+script(type='text/ng-template', id='modals/death.html')
+  .modal-header
+    h3.text-center=env.t('lostAllHealth')
+  .modal-body
+    .container-fluid(ng-controller='UserCtrl')
+      .col-md-6
+        .hero-stats
+          .meter-label(tooltip=env.t('health'))
+            span.glyphicon.glyphicon-heart
+          .meter.health(tooltip='{{Math.round(user.stats.hp * 100) / 100}}')
+            .bar(ng-style='{"width": Shared.percent(user.stats.hp, Shared.maxHealth)+"%"}')
+            span.meter-text.value
+              | {{Math.ceil(user.stats.hp)}} / {{::Shared.maxHealth}}
+        figure.herobox.text-center
```

Comment:
Hero box minus background and with the skin forced to "sleeping". I could
probably figure a way to modularize it, aye!

On Fri, Jul 24, 2015, 4:45 PM Blade Barringer <notifications@github.com>
wrote:

> In website/views/shared/modals/death.jade
> <https://github.com/HabitRPG/habitrpg/pull/5664#discussion_r35468957>:
>
> > +include ../avatar/generated_avatar
> > +
> > +script(type='text/ng-template', id='modals/death.html')
> > +  .modal-header
> > +    h3.text-center=env.t('lostAllHealth')
> > +  .modal-body
> > +    .container-fluid(ng-controller='UserCtrl')
> > +      .col-md-6
> > +        .hero-stats
> > +          .meter-label(tooltip=env.t('health'))
> > +            span.glyphicon.glyphicon-heart
> > +          .meter.health(tooltip='{{Math.round(user.stats.hp * 100) / 100}}')
> > +            .bar(ng-style='{"width": Shared.percent(user.stats.hp, Shared.maxHealth)+"%"}')
> > +            span.meter-text.value
> > +              | {{Math.ceil(user.stats.hp)}} / {{::Shared.maxHealth}}
> > +        figure.herobox.text-center
>
> This is essentially herobox sans background, right? Anyway we can DRY this
> up?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5664/files#r35468957>.
>